spot-oa/oa/flow/flow_conf.json

```diff
@@ -14,25 +14,43 @@
 	    ,"sport":10
 	    ,"dport":11
 	    ,"proto":12
-
 	    ,"ipkt":13
 	    ,"ibyt":14
 	    ,"opkt":15
 	    ,"obyt":16
-
-
 	    ,"score":17
-
     },
-	"column_indexes_filter": [0,8,9,10,11,12,13,14],
+	"column_indexes_filter": [0,8,9,10,11,12,13,14,17],	
 	"flow_score_fields": {
-		"sev": 0,
-		"tstart": 1,
-		"srcIP": 2,
-		"dstIP":3,
-		"sport":4,
-		"dport":5,
-		"ipkt":6,
-		"ibyt":7
+		"sev": 0
+		,"tstart": 1
+		,"srcIP": 2
+		,"dstIP":3
+		,"sport":4
+		,"dport":5
+		,"proto":6
+		,"ipkt":7
+		,"ibyt":8
+		,"lda_score":9
```

Comment:
hi Gabi  --  your changes look very good (I am not sure what I was thinking when I thought I could just blunder in there and change a few names)

but one small thing:

I prefer the name "score" to "lda_score"  since  "score" is a more general thing that comes out of about any anomaly detection system regardless of its internal methods, whereas "lda_score" suggests a dependency on our particular choice of implementation
:)